cdap-ui/app/cdap/components/DataPrep/ColumnActionsDropdown/index.js

```diff
@@ -199,8 +208,8 @@ export default class ColumnActionsDropdown extends Component {
     let tableContainer = document.getElementById('dataprep-table-id');
 
     const tetherOption = {
-      attachment: 'top left',
-      targetAttachment: 'bottom right',
+      attachment: 'top right',
+      targetAttachment: 'bottom left',
```

Comment:
Talked with @elfenheart, `bottom left` target attachment looks better than `bottom right` right now because we have a `margin-left` on the popover. We decided it's ok to keep it like it is.